assets/model_monitoring/components/model_performance/model_performance_compute_metrics/spec.yaml

```diff
@@ -75,6 +75,7 @@ conf:
         - mlflow~=2.8.0
         - azureml-fsspec
         - fsspec~=2023.4.0
+        - azureml-metrics[prompt-flow]
```

Comment:
Yes. Ideally, we should not have this as we already have azureml-metrics in the pip. But it is not working anymore and the failed e2e test case shows that it requires azureml-metrics[prompt-flow], I do not think it is the perfect fix for the failure of e2e, but it would unblock our e2e test in the pipeline quickly. In the meantime, the IDC team is looking into the azureml-metrics to fix it in the right way. Hopefully they can make a fix and we can remove this in the future.